test/services/password_reseter_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Looseleaf.PasswordResetterTest do
+  use Looseleaf.ModelCase, async: true
+  use Bamboo.Test
+  import Looseleaf.Factory
+  alias Looseleaf.Repo
+
+  describe "reset_password(user, repo)" do
+    test "it sets a token to edit a password" do
+      user = insert(:user)
+
+      {:ok, user, _email} = Looseleaf.PasswordResetter.reset_password(user.email, Repo)
+
+      assert user.password_reset_token != nil
```

Comment:
You can just `assert user.password_reset_token` here or better possibly to use `refute user.password_reset_token == nil` if you're into that kind of thing